src/Compiler/Driver/CompilerConfig.fsi

```diff
@@ -926,7 +926,7 @@ val TryResolveFileUsingPaths: paths: string seq * m: range * fileName: string ->
 
 val ResolveFileUsingPaths: paths: string seq * m: range * fileName: string -> string
 
-val GetWarningNumber: m: range * warningNumber: string * prefixSupported: bool -> int option
+val GetWarningNumber: m: range * numStr: string * langVersion: LanguageVersion * isCompilerOption: bool -> int option
```

Comment:
Please replace the isCompilerOption boolean with a 2-case DU (e.g. type WarningNumberSource = CommandLineOption | HashDirective).

I know the compiler uses bool flags elsewhere, but we want to stop adding more.

Reply:
Done